src/sorcha/modules/PPConfigParser.py

```diff
@@ -639,6 +639,27 @@ def PPConfigFileParser(configfile, survey_name):
     else:
         config_dict["rng_seed"] = None
 
+    try:
+        config_dict["lightcurve"] = config.getboolean("EXPERT", "lightcurve", fallback=False)
+    except ValueError:
+        pplogger.error(
+            "ERROR: could not parse value for lightcurve as a boolean. Check formatting and try again."
+        )
+        sys.exit(
+            "ERROR: could not parse value for lightcurve as a boolean. Check formatting and try again."
+        )
+
+    config_dict["lc_model"] = config.get("EXPERT", "lc_model", fallback=None)
+    config_dict["lc_model"] = None if config_dict["lc_model"] == "None" else config_dict["lc_model"]
```

Comment:
I don't think I understand what this line is doing. Is the logic here necessary only because `config_dict["lc_model"]` can have the string value "None" instead of the python `None`?